spec/models/comment_spec.rb

```diff
@@ -247,6 +247,18 @@
     end
   end
 
+  context "when callbacks are triggered after create" do
+    it "enqueue a worker to create the first reaction" do
+      Sidekiq::Testing.fake!
```

Comment:
This is actually [the default mode](https://github.com/mperham/sidekiq/wiki/Testing#setup) for Sidekiq testing so you should be able to remove this line.

Reply:
Hello @msarit good catch :), already removed